tests/tt_metal/tt_metal/perf_microbenchmark/routing/test_common.hpp

```diff
@@ -4,7 +4,7 @@
 
 #pragma once
 
-#include "tt_metal/third_party/json/json.hpp"
+#include <nlohmann/json.hpp>
```

Comment:
@patrickroberts @yan-zaretskiy 
So this is our preferred way of referring to external libraries that we pull in through any dependency management system (in this case, CPM)? So it's clear it's external?

Reply:
external lib ideally you include via `<>`.
things from your project you include via `""`